java/org/apache/juli/FileHandler.java

```diff
@@ -393,6 +430,9 @@ private void configure() {
         if (suffix == null) {
             suffix = getProperty(className + ".suffix", ".log");
         }
+		if (compress == null) {
+			compress = Boolean.valueOf(getProperty(className + ".compress", "true"));
+		}
 
```

Comment:
It looks like some tabs have crept in here. Tomcat uses 4 spaces for idents. I recommend enabling Checkstyle for you build as it will catch this sort of thing,